<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>641</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:black;
display:flex;
flex-direction: column;
</string>
    </property>
    <widget class="QWidget" name="widget565" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>601</width>
       <height>781</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: black</string>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>581</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="functionInput">
           <property name="minimumSize">
            <size>
             <width>434</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #0f0f0f;
border-width: 2px;

border-style: outset;
border-radius: 10px;
border-color: red</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="maxLength">
            <number>20</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
           <property name="placeholderText">
            <string>f(x)</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="xMinInput">
           <property name="styleSheet">
            <string notr="true">border-color:red;
border-top: none;
border-left:none;
border-right:none;

border-width: 2px;
border-style: outset</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">color:white;
font-weight: bold;
font-size: 20px</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&amp;lt; X &amp;lt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="xMaxInput">
           <property name="styleSheet">
            <string notr="true">border-color:red;
border-top: none;
border-left:none;
border-right:none;

border-width: 2px;
border-style: outset</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="plotButton">
           <property name="minimumSize">
            <size>
             <width>87</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 255, 100, 255), stop:1 rgba(255, 0, 0, 255));

border-color:red;
border-style: outset;
border-width:0.5px;
border-radius: 5px;

padding: 5px;
color: black;
font-weight: bold;</string>
           </property>
           <property name="text">
            <string>Plot</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>0</y>
        <width>81</width>
        <height>70</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
font-weight: bold;
font-size: 20px</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Dezmoz&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>571</width>
        <height>241</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="plotArea"/>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
